tutorials/quickstart/tutorial.md

```diff
@@ -0,0 +1,87 @@
+
+
+## Protection from transient errors
+
+Have you ever had a business process get interrupted by a transient error like a database deadlock? Usually this leaves a system in an inconsistent state, like the order persisted to the database but not yet submitted to the payment processor. As a developer, you might have to dive directly into the database like a forensic analysis, trying to figure out where the process went wrong, and how to manually jump-start it so that the process can complete.
+
+With NServiceBus, this doesn't have to happen. Each message handler will automatically retry processing messages if an exception is thrown, so transient failures like database deadlocks are handled gracefully, without manual intervention by a developer.
+
+Let's introduce a transient failure into the **Sales** endpoint and see this in action.
+
+1. In the **Sales** endpoint, locate and open the **PlaceOrderHandler.cs** file.
+1. Uncomment the code inside the **ThrowTransientException** region shown here. This will cause an exception to be thrown 20% of the time a message is processed:
+
+snippet: ThrowTransientException
+
+3. Start the solution without debugging (Ctrl+F5), or alternatively, start the solution and then select **Detach All** in the **Debug** menu. This will make it easier to observe exceptions occurring without being interrupted by Visual Studio's Exception Assistant.
+3. In the **ClientUI** window, slowly start sending one message at a time by pressing `P`, and be sure to watch the **Sales** window.
+
+As you watch the **Sales** window, 80% of the messages will go through as normal, but when an exception occurs, the output will be different:
+
+    INFO  NServiceBus.RecoverabilityExecutor Immediate Retry is going to retry message '43400b29-c235-471f-ab4f-a7760145ea88' because of an exception:
+    System.Exception: Oops
+       at <long stack trace>
+    INFO  Sales.PlaceOrderHandler Received PlaceOrder, OrderId = e1d86cb9-c393-475b-9be0-5407e9e529e0
+
+By using automatic retries, you can avoid losing data or having your system left in an inconsistent state on account of a stray database deadlock. No more spelunking through the database trying to fix business processes gone wrong!
+
+
+## Easy to extend
+
+Stub: Instructions for adding Shipping service with subscriber for OrderPlaced
+
+## Summary
+
+Summarize, link to CTA
```

Comment:
not sure if it's a good idea to link to blog from here, but "but all my errors are severe" would be excellent here